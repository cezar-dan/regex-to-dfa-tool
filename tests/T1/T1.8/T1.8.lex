 
SPACE
0
0,' ',1
1,' ',2
1

ab
ABSTAR
0
0,'a',1
0,'b',2
1,'a',2
1,'b',3
3,'a',1
3,'b',2
3 0

abc
ABPLUSC
0
0,'a',1
0,'b',2
0,'c',2
1,'a',2
1,'b',3
1,'c',2
3,'a',1
3,'b',2
3,'c',4
4,'a',2
4,'b',2
4,'c',4
4 3

bc
BCBC
0
0,'b',1
0,'c',2
1,'b',2
1,'c',3
3,'b',4
3,'c',2
4,'b',2
4,'c',5
5,'b',2
5,'c',2
5

bc
CBSAR
0
0,'b',1
0,'c',2
2,'b',3
2,'c',1
3,'b',1
3,'c',2
3 0

bc
BORCS
0
0,'b',1
0,'c',2
1,'b',1
1,'c',2
2,'b',1
2,'c',2
2 1 0

abde
ABDORE
0
0,'a',1
0,'b',2
0,'d',2
0,'e',3
1,'a',2
1,'b',4
1,'d',2
1,'e',2
3,'a',2
3,'b',2
3,'d',2
3,'e',2
4,'a',2
4,'b',2
4,'d',5
4,'e',2
5,'a',2
5,'b',2
5,'d',2
5,'e',2
3 5

abd
ASTARBD
0
0,'a',1
0,'b',2
0,'d',3
1,'a',1
1,'b',2
1,'d',3
2,'a',3
2,'b',3
2,'d',4
4,'a',3
4,'b',3
4,'d',3
4

ef
EFSTAR
0
0,'e',1
0,'f',2
1,'e',2
1,'f',3
3,'e',2
3,'f',3
3 1

c
C
0
0,'c',1
1,'c',2
1