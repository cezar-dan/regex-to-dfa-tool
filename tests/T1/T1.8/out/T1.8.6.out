ABSTAR ab
BORCS bcbccb
SPACE  
CBSAR cbcbcb
SPACE  
BORCS bcbcbc
SPACE  
BORCS c